src/scenes/Event/store.js

```diff
@@ -72,7 +74,7 @@ export default class EventStore {
   }
   // For Oracle only
   @computed get unconfirmed() {
-    return this.topicAddress === 'null' && this.address === 'null';
+    return _.isUndefined(this.topicAddress) && _.isUndefined(this.address);
```

Comment:
are you sure this works? I believe the `topicAddress` and `address` are actually a string that contains `null`

Reply:
we used to have  `null` in our path, but now we don't have it, so the destruct of `topicAddress` and `address` cannot get `null`

Reply:
oh right, duhhhh lol